src/components/common/GProfiler/utils.ts

```diff
@@ -0,0 +1,141 @@
+import {
```

Comment:
`utils` is not a very descriptive filename. Maybe the link create and check functions could go in a `link.ts` and `getMetricTable`, `getAvailableSortBy` could go in `model.ts`?